packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -29,18 +37,53 @@ const validateConfig = (config: ArithmeticConfig) => {
   });
 };
 
+const analyticsToAnalyticalEvents = (analytics: Analytic[]): AnalyticalEvent[] => {
```

Comment:
I think it's worth moving this to `data-broker` and thus standardising the concept of `AnalyticalEvents`, but would like your opinion on that

Reply:
That's a very good point you make here! So this format here does not necessarily map to a human-conceived "event": it's just a convenient aggregation of values for a specific space/time frame. 

If we agree that this is useful, we should then find an appropriate name (you can see I struggled with that)

Reply:
How about `AnalyticCluster`?

```suggestion
const clusterAnalyticsByPeriodAndOrgUnit = (analytics: Analytic[]): AnalyticCluster[] => {
```

I'd be nervous to introduce that to `data-broker` as anything other than a util though, because without taking the proper time to roll out standardised terminology and socialise it to the team, we risk adding more confusion to the mix.

Reply:
I love `AnalyticCluster`!!

Yeah we shouldn't introduce this in any other way rather than a util for now. Which opens the possibility of using this concept again in the future, without adding significant complexity at this stage